playground/src/main/java/org/navgurukul/playground/ui/PlaygroundFragment.kt

```diff
@@ -16,6 +16,10 @@ class PlaygroundFragment : BaseFragment() {
 
     private val viewModel: PlaygroundViewModel by viewModel()
     private val navigator: MerakiNavigator by inject()
+    private val TYPE = "practicetyping";
+    private val TYPING_CONTENT = arrayListOf("a","b","c","d","e","f","g",
```

Comment:
This logic should be contained inside typing app module, playground should not know internal working of typing app